src/main/scala/nl/knaw/dans/easy/multideposit/actions/AddPropertiesToDeposit.scala

```diff
@@ -66,17 +111,20 @@ object AddPropertiesToDeposit {
       .getOrElse(Failure(ActionException(row, """The column "DEPOSITOR_ID" is not present""")))
   }
 
-  def writeProperties(row: Int, datasetID: DatasetID, dataset: Dataset)(implicit settings: Settings): Try[Unit] = {
-    val props = new Properties
+  def writeProperties(row: Int, datasetID: DatasetID, dataset: Dataset, emailaddress: String)(implicit settings: Settings): Try[Unit] = {
+    val props = new Properties {
+      // Make sure we get sorted output, which is better readable than random
+      override def keys(): util.Enumeration[AnyRef] = Collections.enumeration(new util.TreeSet[Object](super.keySet()))
+    }
 
-    addProperties(props, dataset)
+    addProperties(props, dataset, emailaddress)
       .flatMap(_ => Using.fileWriter(encoding)(outputPropertiesFile(datasetID)).map(out => props.store(out, "")).tried)
       .recoverWith {
         case NonFatal(e) => Failure(ActionException(row, s"Could not write properties to file: $e", e))
       }
   }
 
-  def addProperties(properties: Properties, dataset: Dataset): Try[Unit] = {
+  def addProperties(properties: Properties, dataset: Dataset, emailaddress: String)(implicit settings: Settings): Try[Unit] = {
```

Comment:
Maybe you should give the datamanager's id as an argument instead of giving the whole `Settings` object here?